rockstar/rockstar.js

```diff
@@ -339,6 +341,38 @@
         });
     }
     
+    function directoryGroup() {
+        var groupId = location.pathname.split("/")[3];
+        var group;
+        getJSON(`/api/v1/groups/${groupId}`).then(aGroup => {
+            group = aGroup;
+            $(".subheader").html(`${e(group.profile.name)}`);

```

Comment:
there is no element with this css class. it exists for Users, but not for Groups

Reply:
👍🏼 makes sense. I'll delete this line and resubmit.

Anything else?